server/config/config.go

```diff
@@ -954,8 +954,14 @@ type ReplicationConfig struct {
 	// StrictlyMatchLabel strictly checks if the label of TiKV is matched with LocationLabels.
 	StrictlyMatchLabel bool `toml:"strictly-match-label" json:"strictly-match-label,string"`
 
-	// When PlacementRules feature is enabled. MaxReplicas and LocationLabels are not uesd any more.
+	// When PlacementRules feature is enabled. MaxReplicas and LocationLabels are not used any more.
 	EnablePlacementRules bool `toml:"enable-placement-rules" json:"enable-placement-rules,string"`
+
+	// IsolationLabel is used to isolate replicas explicitly and forcibly.
+	// For example, with isolation-label = zone, PD ensure that all replicas be placed in different zones.
+	// Even if a zone is down, PD will not try to make up replicas in other zone
+	// because other zones already have replicas on it.
+	IsolationLabel string `toml:"isolation-label" json:"isolation-label"`
```

Comment:
What's the default value of the `IsolationLabel`?  Does it always be enabled or have a switch to control ?

Reply:
Also I think we can list the valid option for this configuration.

Reply:
Maybe `isolation-level` is better?

Reply:
`IsolationLabel` should be one of the `LocationLabels`. If it's empty, we will think the strict isolation level is off. But if we rename it to `isolation-level`, maybe its value should be the index of `LocationLabels`? For example, we may have a configuration like this:

```
LocationLabels: ["zone", "rack", "host"]

IsolationLabel: "zone"
// Or IsolationLevel: 0
```

What's your opinion?

Reply:
I tend to make the configuration in following format which is obvious for the users even they have little knowledge about the pd isolation scheduling strategy.

```sh
LocationLabels: ["zone", "rack", "host"]

IsolationLevel: "zone"
```

Reply:
Nice idea.